### PR TITLE
Release 7.3.0

### PR DESCRIPTION
**Pull request by crow**
### What do these changes do?
Release 7.3.0
- Update iOS SDK to 10.0.0

### How did you verify these changes?
Manually tested the iOS sample.

### CODE REVIEW

**Comment by crow** [comment]
this is failing due to a cloning step failure, need more git bandwidth

**Comment by rlepinski** [comment]
👍

**Comment by rlepinski** [comment]
`
         <source-file framework="true" src="src/ios/Airship/libUAirship-9.4.0.a"/>` in the plugin.xml needs to be updated.

File: plugin.xml.orig [hunk 1]
```diff
@@ -0,0 +1,372 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (BrianBatchelder) [inline plugin.xml.orig]: Please remove this file.

**Pull request merged**